src/extension/importer/core/src/test/java/org/geoserver/importer/ImporterTestSupport.java

```diff
@@ -74,6 +74,11 @@ protected void onSetUp(SystemTestData testData) throws Exception {
         namespaces.put("wms", "http://www.opengis.net/wms");
         
         XMLUnit.setXpathNamespaceContext(new SimpleNamespaceContext(namespaces));
+        
+        // add limits properties file
```

Comment:
Limits?

Reply:
The test should also check that the imported data actually ends up in that directory (either here or in a REST test)

Reply:
The comment is wrong, will change it.

I'll add REST test also.